<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetEstoques</class>
 <widget class="QWidget" name="WidgetEstoques">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonFollowup">
       <property name="text">
        <string>Followup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRelatorio">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Gerar relatório</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxRelatorio">
       <property name="title">
        <string>Relatório contábil</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <widget class="QDateEdit" name="dateEditMes">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonRelatorioContabil">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Gerar relatório data</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonExportarNCM">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Exportar NCMs</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxFiltros">
       <property name="title">
        <string>Filtros</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioButtonMaior">
          <property name="text">
           <string>Quant. &gt; 0</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonEstoqueZerado">
          <property name="text">
           <string>Quant. = 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonEstoqueContabil">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Estoque contábil</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelAviso">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Verificar com o Dept. de Compras a disponibilidade do estoque antes de vender!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelBuscar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Buscar:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="LineEdit" name="lineEditBusca">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Produto/Cód. Com./Fornecedor/Estoque</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="TableView" name="tableEstoque">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/lineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonFollowup</tabstop>
  <tabstop>dateEditMes</tabstop>
  <tabstop>pushButtonRelatorioContabil</tabstop>
  <tabstop>radioButtonMaior</tabstop>
  <tabstop>radioButtonEstoqueZerado</tabstop>
  <tabstop>radioButtonEstoqueContabil</tabstop>
  <tabstop>tableEstoque</tabstop>
  <tabstop>lineEditBusca</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
